wire-ios/Wire-iOS/Resources/Settings.bundle/com.mono0926.LicensePlist.plist

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>PreferenceSpecifiers</key>
+	<array>
+		<dict>
+			<key>Type</key>
+			<string>PSGroupSpecifier</string>
+			<key>Title</key>
+			<string>No licences have been generated. If you believe that this is not correct, please send a report.</string>
```

Comment:
Just a default string for debug builds as the licenses are generated and included through CI.